mapping.py

```diff
@@ -0,0 +1,94 @@
+from fuzzywuzzy import fuzz #added
+import stream_deepspeech
+from timeit import default_timer as timer
+import json
+
+def get_match(substring):
+    """Returns the best match of a given substring, otherwise, returns no match"""
+    matches = [(correctionJSON()[key], fuzz.ratio(substring, key)) for key in correctionJSON().keys()
+               if fuzz.ratio(substring, key) > 70]
```

Comment:
out of  curiosity, do you know what the fuzz ratio is for something like "you nah leg ah quadra" and "Una Legua Cuadrada"?

Reply:
Ahhhh that's a good question! They're not exactly similar in the way they are spelled, but they have a similar pronunciation. The JSON file contains possible ways that someone can pronounce a name, and each of those pronunciations is mapped to their respective correct names. (e.g. "you nah leg ah quadra" : "Una Legua Cuadrada") The get_match function would try to find a way of pronunciation that is similar to the audio input. For example, if the audio input is "you ah lay ah quadra," get_match would find "you nah leg ah quadra." Since "you nah leg ah quadra" is a key for the value "Una Legua Cuadrada," the function get_match would find Una Legua Cuadrada as the best match.